apps/test/applabTest.js

```diff
@@ -8,13 +8,14 @@ window.$ = $;
 window.jQuery = window.$;
 window.React = React;
 
+// used in design mode
 window.Applab = {
   appWidth: 320,
   appHeight: 480
 };
 
-var AppLab = require('@cdo/apps/applab/applab');
-var designMode = require('@cdo/apps/applab/designMode');
+var Applab = require('@cdo/apps/Applab/Applab');
```

Comment:
looks like there may have been some overzealous find/replace on the strings that got upcased here